### DISCUSSION TITLE
ACF- Location Settings Not Saving

### DISCUSSION THREAD

**Discussion by shandyw** [flyntwp/flynt#550] [General]
Hello,
Has anyone else experienced issues with ACF (GUI) not saving when changing location settings? 
Everything else seems to be working ok- but I cannot create a CPT page specific field because the location settings reverts to "post" on saving. I've tried 
-removing components
-removing /inc function files
-disabling ALL plugins (except ACF)
-checked php max memory
-checked DB wp_options option_name max char limits
Does anyone have any other ideas? This is driving me nuts. 
thx!

**Comment by timohubois** [comment]
@shandyw this is maybe related to this: https://github.com/flyntwp/flynt/pull/545

Can you try to use the updated “theContentFix.php” file: https://github.com/flyntwp/flynt/blob/543-flynt-v21-cannot-correctly-save-acf-field-groups/inc/theContentFix.php

**Reply by shandyw** [reply]
THANK YOU SO MUCH!!! That fixed it.
